course5/homework1/homework1.cc

```diff
@@ -1 +1,8 @@
-// your code goes here
+#include <iostream>
+
+int main(int argc, char** argv) {
+  for (size_t i = 0; i < argc; i++) {
```

Comment:
You are right. size_t to int does the trick. cmake/make did not yield a warning. Shouldn't the following make it display warnings: set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14") ?